README.md

```diff
@@ -5,29 +5,11 @@ This repository contains official Helm charts that are maintained by [Aiven](htt
 ## Adding the repository
 
 ```bash
-helm repo add aiven https://aiven-charts.github.io && helm repo update
+helm repo add aiven https://aiven.github.io/aiven-charts && helm repo update
 ```
 
 ## Installing a chart from the repository
 
-### Installing the aiven-operator-crds chart
+### Installing the aiven-operator
 
-```bash
-helm install aiven/aiven-operator-crds --generate-name
-```
-
-### Installing the aiven-operator chart
-
-Before installing the `aiven-operator` chart, please make sure to have the CRDs chart installed.
-
-Installation with webhooks enabled ( requires a cert-manager deployment in the cluster ):
-
-```bash
-helm install aiven/aiven-operator --generate-name
-```
-
-Without webhooks enabled:
-
-```bash
-helm install aiven/aiven-operator --generate-name --set webhooks.enabled=false
-```
+Please see the [installation instructions](https://aiven.github.io/aiven-operator/docs/installation/helm/) in the [Aiven Operator for Kubernetes](https://github.com/aiven/aiven-operator) documentation.
```

Comment:
It is a bit strange to go the `[installation instructions](https://aiven.github.io/aiven-operator/docs/installation/helm/)` where there are no instructions actually but a link back to this repo README. Can we optimize it?

Reply:
This will be fixed after the https://github.com/aiven/aiven-operator/pull/158 is merged

Reply:
the issue with the version is addressed in this PR: charts/aiven-operator/Chart.yaml and charts/aiven-operator-crds/Chart.yaml